maps/includes/base_shutdown.lua

```diff
@@ -88,6 +88,8 @@ function button_common:allowed( allowed_entity )
 	if IsPlayer( allowed_entity ) then
 		local player = CastToPlayer( allowed_entity )
 		if player:GetTeamId() == self.team and player:IsAlive() then
+			player:AddFortPoints(100, "disabled security")

```

Comment:
This should be a localized string (`#FF_FORTPOINTS_DISABLE_SECURITY` for example).

See https://github.com/fortressforever/fortressforever-scripts/blob/master/maps/includes/base_teamplay.lua#L431